product-service/helpers/validateProduct.js

```diff
@@ -0,0 +1,25 @@
+import Schema from 'validate';
+
+const productSchema = new Schema({
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+  },
+  price: {
+    type: Number,
+    required: true,
+  },
+  count: {
+    type: Number,
+    required: true,
+  },
+});
+
+export const validateProduct = (product) => {
+  const errors = productSchema.validate(product);
```

Comment:
You don't validate on negative integers. I have created products with invalid price and count:
![image](https://user-images.githubusercontent.com/70973590/181625740-f7bd5b93-1583-4e20-ae0b-8a23506127a1.png)

Reply:
Done.